example/main.go

```diff
@@ -11,6 +11,7 @@ package main
 // @Server https://app.launchdarkly.com
 // @Security ApiKey read write
 // @SecurityScheme ApiKey apiKey header Authorization
+// @RenamePackage "example" ""
```

Comment:
Nit. Instead of "RenamePackage" consider "PackageName" or "PackageAlias".

This is an imperative instruction, where I think we'd prefer our annotations to be declarative.

Reply:
I had `PackageName` originally but thought it may be confusing without more context. I'll change to `PackageAlias` that seems clear on intent.

Reply:
Agreed that `PackageName` is confusing, and doesn't by itself tell you what the annotation is for. `PackageAlias` is an excellent choice 👍